dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -101,6 +102,7 @@ public class ConnectorArguments extends DefaultArguments {
   public static final String OPT_ORACLE_SERVICE = "oracle-service";
 
   public static final String OPT_QUERY_LOG_DAYS = "query-log-days";
+  public static final String OPT_QUERY_LOG_INTERVAL = "query-log-interval";
```

Comment:
I would slightly suggest to rename to `query-log-rotation-interval`. It's longer, but more descriptive.

Reply:
Renamed to `query-log-rotation-frequency` as per the cmd argument text